spec/reek/smells/prima_donna_method_spec.rb

```diff
@@ -3,28 +3,19 @@
 require_relative 'smell_detector_shared'
 
 RSpec.describe Reek::Smells::PrimaDonnaMethod do
+  it 'reports the right values' do
+    src = 'class C; def m!; end; end'
+    expect(src).to reek_of :PrimaDonnaMethod,
+                           lines: [1],
+                           message: 'has prima donna method `m!`',
+                           name: :m!
+  end
```

Comment:
Yes, let's do that right away after #998 
I guess we can then kill 1000's of LoC.